### PR TITLE
DOC/STY: update instrument docstrings and adjust style post 2.2 release

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #486.  Pull 1 of ?

- Updates doctrings
- Moves imports to top
- Removes comments and unused functions left over from templates

## Type of change

Please delete options that are not relevant.

- Style fix (non-breaking change which fixes an issue)
- This change is a documentation update

# How Has This Been Tested?

reading the supported instrument list after using `make html` to ensure the docstrings are formatted properly

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
`

### CODE REVIEW

File: pysat/instruments/cosmic_gps.py [hunk 1]
```diff
@@ -42,23 +42,27 @@
...
 
 """
 
-from __future__ import print_function
-from __future__ import absolute_import
 import datetime as dt
+import logging
 import os
+import requests
+from requests.auth import HTTPBasicAuth
```
Comment (aburrell) [inline pysat/instruments/cosmic_gps.py:56]: Stylistically, is the second import really needed?

File: pysat/instruments/omni_hro.py [hunk 2]
```diff
@@ -281,7 +281,9 @@ def calculate_dayside_reconnection(inst):
     Notes
     --------
```
Comment (aburrell) [inline pysat/instruments/omni_hro.py:282]:
```suggestion
    Note
    -----
```

File: pysat/instruments/sw_dst.py [hunk 3]
```diff
@@ -182,30 +187,22 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
...
         Path to data directory.  If None is specified, the value previously
         set in Instrument.files.data_path is used.  (default=None)
 
-    Returns
-    --------
-    Void : (NoneType)
-        data downloaded to disk, if available.
-
     Notes
     -----
```
Comment (aburrell) [inline pysat/instruments/sw_dst.py:201]:
```suggestion
    Note
    -----
```

File: pysat/instruments/sw_f107.py [hunk 4]
```diff
@@ -42,16 +43,21 @@
 
 """
 
-import os
 import datetime as dt
+import ftplib
+from ftplib import FTP
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:48]: Again, suggest just using the top import

File: pysat/instruments/sw_f107.py [hunk 5]
```diff
@@ -85,18 +91,18 @@ def load(fnames, tag=None, sat_id=None):
...
     Returns
     ---------
-    data : (pandas.DataFrame)
+    data : pandas.DataFrame
         Object containing satellite data
-    meta : (pysat.Meta)
+    meta : pysat.Meta
         Object containing metadata such as column names and units
 
     Notes
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:108]:
```suggestion
    Note
```

File: pysat/instruments/sw_f107.py [hunk 6]
```diff
@@ -179,22 +185,22 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None):
...
         User specified file format.  If None is specified, the default
         formats associated with the supplied tags are used. (default=None)
 
     Returns
     --------
-    pysat.Files.from_os : (pysat._files.Files)
+    pysat.Files.from_os : pysat._files.Files
         A class containing the verified available files
 
     Notes
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:206]:
```suggestion
    Note
```

File: pysat/instruments/sw_kp.py [hunk 7]
```diff
@@ -50,17 +51,26 @@
...
+Custom Functions
+----------------
+filter_geoquiet
+    Filters pysat.Instrument data for given time after Kp drops below gate.
+
 """
 
 import datetime as dt
+import ftplib
+from ftplib import FTP
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:63]: See previous

File: pysat/instruments/sw_kp.py [hunk 8]
```diff
@@ -181,24 +191,25 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None):
...
 
     Returns
     --------
-    pysat.Files.from_os : (pysat._files.Files)
+    pysat.Files.from_os : pysat._files.Files
         A class containing the verified available files
 
+
     Notes
     -----
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:214]:
```suggestion
    Note
    ----
```

File: pysat/instruments/sw_kp.py [hunk 9]
```diff
@@ -421,11 +423,6 @@ def filter_geoquiet(sat, maxKp=None, filterTime=None, kpData=None,
     kp_inst : pysat.Instrument (optional)
         Kp pysat.Instrument object ready to load Kp data.Overrides kpData.
 
-    Returns
-    -------
-    None : NoneType
-        sat Instrument object modified in place
-
     Notes
     -----
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:427]:
```suggestion
    Note
    ----
```

File: pysat/instruments/timed_saber.py [hunk 10]
```diff
@@ -185,18 +105,10 @@ def clean(inst):
 
     Parameters
     -----------
```
Comment (aburrell) [inline pysat/instruments/timed_saber.py:107]:
```suggestion
    ----------
```

File: pysat/instruments/cnofs_ivm.py [hunk 11]
```diff
@@ -99,17 +98,12 @@ def clean(inst):
 
     Parameters
     -----------
-    inst : (pysat.Instrument)
+    inst : pysat.Instrument
```
Comment (rstoneback) [inline pysat/instruments/cnofs_ivm.py:101]: I can't find a way to insert a comment or suggestion on unchanged line.
for default method

```
"""Apply C/NOFS IVM default attributes

The sample rate for loaded data is attached at inst.sample_rate
before any attached custom methods are executed.

Parameters
----------
ivm : pysat.Instrument
    C/NOFS IVM Instrument object

 """

```

```suggestion
    inst : pysat.Instrument
```

File: pysat/instruments/demeter_iap.py [hunk 12]
```diff
@@ -15,35 +15,45 @@
...
+::
+
     import pysat
     demeter = pysat.Instrument('demeter', 'iap', 'survey', clean_level='none')
     demeter.load(2009,363)
 
-"""
 
-from __future__ import print_function, absolute_import
+Custom Functions
```
Comment (rstoneback) [inline pysat/instruments/demeter_iap.py:38]: nice

File: pysat/instruments/sw_kp.py [hunk 13]
```diff
@@ -86,18 +95,18 @@ def load(fnames, tag=None, sat_id=None):
...
     Returns
     ---------
-    data : (pandas.DataFrame)
+    data : pandas.DataFrame
         Object containing satellite data
-    meta : (pysat.Meta)
+    meta : pysat.Meta
         Object containing metadata such as column names and units
 
     Notes
```
Comment (rstoneback) [inline pysat/instruments/sw_kp.py:112]:
```suggestion
    Note
    ----
```

**Comment by jklenzing** [comment]
Thanks @aburrell and @rstoneback!  All updates should be applied.

File: pysat/instruments/sw_dst.py [hunk 14]
```diff
@@ -27,8 +27,12 @@
 """
 
 import datetime as dt
+import ftplib
+from ftplib import FTP
```
Comment (aburrell) [inline pysat/instruments/sw_dst.py:31]: Another instance of effectively a double import.

File: pysat/instruments/sw_kp.py [hunk 15]
```diff
@@ -86,21 +95,22 @@ def load(fnames, tag=None, sat_id=None):
...
+    data : pandas.DataFrame
         Object containing satellite data
-    meta : (pysat.Meta)
+    meta : pysat.Meta
         Object containing metadata such as column names and units
 
-    Notes
+    Note
+    ----
     -----
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:114]:
```suggestion
```

File: pysat/instruments/sw_kp.py [hunk 16]
```diff
@@ -181,26 +191,27 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None):
...
+    data_path : string or NoneType
         Path to data directory.  If None is specified, the value previously
         set in Instrument.files.data_path is used.  (default=None)
-    format_str : (string or NoneType)
+    format_str : string or NoneType
         User specified file format.  If None is specified, the default
         formats associated with the supplied tags are used. (default=None)
 
     Returns
     --------
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:208]:
```suggestion
    -------
```

File: pysat/instruments/sw_dst.py (hunk 14 above)
Comment (jklenzing) [reply pysat/instruments/sw_dst.py]: fixed!

**Pull request merged**